CHANGELOG.md

```diff
@@ -63,6 +43,12 @@ Special thanks to external contributors on this release: @JayT106, @bipulprasad,
   - [abci/counter] [\#6684](https://github.com/tendermint/tendermint/pull/6684) Delete counter example app
 
 - Go API
+  - [crypto/armor]: [\#6963](https://github.com/tendermint/tendermint/pull/6963) remove package which is unused, and based on
+    deprecated fundamentals. Downstream users should maintain this
+    library. (@tychoish)
+  - [state] [store] [proxy] [rpc/core]: [\#6937](https://github.com/tendermint/tendermint/pull/6937) move packages to
```

Comment:
Consolidate with line 55.

Reply:
Oops, sorry, I missed this comment. It's a good idea, though, I'll do that separately.